File: src/main/java/it/contrader/automative/model/Appuntamento.java
package it.contrader.automative.model;

import java.io.Serializable;
import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.OneToOne;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor

@Entity
@Table(name = "appuntamento")
public class Appuntamento implements Serializable {

	@Id
	private int id;
	
	
	@ManyToOne(fetch=FetchType.LAZY)
	@JoinColumn(name = "idUtente")
	private Utente utente;
	
	@ManyToOne(fetch=FetchType.LAZY)
	@JoinColumn(name = "idOfficina")
	private Officina officina;
	
	@Column
	private Date Data;
	
	@Column
	private String Ora;

	@Column 
	private String Dettagli;
	
	@Column
	private int Stato;
	
	@Column 
	private String Risposta;
	
	
}

File: FrontAutomotiveAngular5/src/app/models/Appuntamento.ts
import { Officina } from './Officina';
import { Utente } from './Utente'; 

export class Appuntamento {
    private id: number;
    private data: string;
    private dettagli: string;
    private ora: string;
    private risposta: string;
    private stato: number;
    private officina: Officina;
    private utente: Utente;


	constructor($id: number, $data: string, $dettagli: string, $ora: string, $risposta: string, $stato: number, $officina: Officina, $utente: Utente) {
		this.id = $id;
		this.data = $data;
		this.dettagli = $dettagli;
		this.ora = $ora;
		this.risposta = $risposta;
		this.stato = $stato;
		this.officina = $officina;
		this.utente = $utente;
	}
        
}